README.md

```diff
@@ -77,18 +89,18 @@ Contributions are what make the open source community such an amazing place to b
 
 <!-- LICENSE -->
 ## License
-Distributed under the Apache License. See [`LICENSE`](https://github.com/padaiyal/jMavenProjectTemplate/blob/main/LICENSE) for more information.
+Distributed under the Apache License. See [`LICENSE`](https://github.com/padaiyal/jI18n/blob/main/LICENSE) for more information.
 
 
 <!-- MARKDOWN LINKS & IMAGES -->
 <!-- https://www.markdownguide.org/basic-syntax/#reference-style-links -->
-[contributors-shield]: https://img.shields.io/github/contributors/padaiyal/jMavenProjectTemplate.svg?style=for-the-badge
-[contributors-url]: https://github.com/padaiyal/jMavenProjectTemplate/graphs/contributors
-[forks-shield]: https://img.shields.io/github/forks/padaiyal/jMavenProjectTemplate.svg?style=for-the-badge
-[forks-url]: https://github.com/padaiyal/jMavenProjectTemplate/network/members
-[stars-shield]: https://img.shields.io/github/stars/padaiyal/jMavenProjectTemplate.svg?style=for-the-badge
-[stars-url]: https://github.com/padaiyal/jMavenProjectTemplate/stargazers
-[issues-shield]: https://img.shields.io/github/issues/padaiyal/jMavenProjectTemplate.svg?style=for-the-badge
-[issues-url]: https://github.com/padaiyal/jMavenProjectTemplate/issues
-[license-shield]: https://img.shields.io/github/license/padaiyal/jMavenProjectTemplate.svg?style=for-the-badge
-[license-url]: https://github.com/padaiyal/jMavenProjectTemplate/blob/master/LICENSE.txt
+[contributors-shield]: https://img.shields.io/github/contributors/padaiyal/jI18n.svg?style=for-the-badge
+[contributors-url]: https://github.com/padaiyal/jI18n/graphs/contributors
+[forks-shield]: https://img.shields.io/github/forks/padaiyal/jI18n.svg?style=for-the-badge
+[forks-url]: https://github.com/padaiyal/jI18n/network/members
+[stars-shield]: https://img.shields.io/github/stars/padaiyal/jI18n.svg?style=for-the-badge
+[stars-url]: https://github.com/padaiyal/jI18n/stargazers
+[issues-shield]: https://img.shields.io/github/issues/padaiyal/jI18n.svg?style=for-the-badge
+[issues-url]: https://github.com/padaiyal/jI18n/issues
+[license-shield]: https://img.shields.io/github/license/padaiyal/jI18n.svg?style=for-the-badge
+[license-url]: https://github.com/padaiyal/jI18n/blob/master/LICENSE.txt
```

Comment:
```suggestion
[license-url]: https://github.com/padaiyal/jI18n/blob/master/LICENSE
```
This is a bug in jPopper, jMavenProjectTemplate, jProperties and this repo.
Please fix it for this repo in this PR and address it in jPopper and jMavenProjectTemplate. I'll take care of jProperties.